test/appium/tests/critical/test_public_chat_browsing.py

```diff
@@ -814,11 +814,10 @@ def test_community_contact_block_unblock_offline(self):
     def test_community_mark_all_messages_as_read(self):
         self.channel_1.click_system_back_button_until_element_is_shown()
         self.home_1.communities_tab.click()
-        if not self.channel_2.chat_message_input.is_element_displayed():
-            self.channel_2.click_system_back_button_until_element_is_shown()
-            self.home_2.communities_tab.click()
-            self.home_2.get_chat(self.community_name, community=True).click()
-            self.community_2.get_channel(self.channel_name).click()
+        self.channel_2.click_system_back_button_until_element_is_shown()
+        self.home_2.communities_tab.click()
+        self.home_2.get_chat(self.community_name, community=True).click()
+        self.community_2.get_channel(self.channel_name).click()
```

Comment:
maybe better to use:
 `self.channel2.jump_to_card_by_text('# %s' % self.channel_name)`
 
 and how affect the test deleting `if` in particular case?

Reply:
Not sure about 'jump-to' because it works not in all cases so I changed it before.
About deleting 'if' clause: previous test ends up in 1-1 chat and here we were waiting for for presence of the inout which is similar for both 1-1 and community. So what this test did before - it sent a message in 1-1 chat instead of a community channel and failed because didn't receive anything in community. That's why I changed it to always navigate to community channel

Reply:
Ah! got it, thanks.
May leave as it is or use `get_to_community_channel_from_home`

Reply:
I'll leave it for now